contracts/MainframeToken.sol

```diff
@@ -47,11 +46,11 @@ contract MainframeToken is ERC827Token, Pausable, Claimable {
 
   // ERC827 Methods
 
-  function transferAndCall(address to, uint256 value, bytes data) public validDestination(to) payable isTradeable returns (bool) {
+  function transferAndCall(address to, uint256 value, bytes data) public payable isTradeable returns (bool) {
     return super.transferAndCall(to, value, data);
   }
 
-  function transferFromAndCall(address from, address to, uint256 value, bytes data) public validDestination(to) payable isTradeable returns (bool) {
+  function transferFromAndCall(address from, address to, uint256 value, bytes data) public payable isTradeable returns (bool) {
```

Comment:
Any reason why we're removing this only from the `AndCall` versions of these functions?

Reply:
Ah, I see where the checks are already happening in `AndCall` versions in `ERC827Token`